test/unit/checks_test.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:convert';
+
+import 'package:github/src/common/model/checks.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Check run', () {
+    test('CheckRun fromJson', () {
+      // This is a truncated version of the response
+      const checkRunJson = '''{
```

Comment:
Please use the official GitHub values provided in the API -- https://docs.github.com/en/rest/reference/checks#get-a-check-run

And add a link to where it is from

Reply:
Sure Casey, I will change this to the official one.